lib/open_api_spex/cast.ex

```diff
@@ -95,4 +121,20 @@ defmodule OpenApiSpex.Cast do
     error = Error.new(ctx, error_args)
     {:error, [error | ctx.errors]}
   end
+
+  def ok(%__MODULE__{value: value}), do: {:ok, value}
+
+  def success(%__MODULE__{schema: schema} = ctx, schema_properties)
+      when is_list(schema_properties) do
+    schema_without_successful_validation_property =
+      Enum.reduce(schema_properties, schema, fn property, schema ->
+        Map.replace!(schema, property, nil)
```

Comment:
I've never used `Map.replace!/3` before. It looks identical to the behavior of `%{schema | key => value}`. Is that correct?